scripts/superdesk/ui/ui.js

```diff
@@ -423,6 +423,13 @@
                                 }
                             });
 
+                            inputField.on('keydown', function(event) {
```

Comment:
you will have to unregister this handler manually, otherwise it will leak memory.
with `scope.$on('$destroy', ...)`